lmi_utils/image_utils/resize_image.py

```diff
@@ -34,7 +34,7 @@ def resize_images(path_imgs, output_imsize, path_out):
         if ratio_in != ratio_out:
             logger.warning(f'file: {im_name}, asepect ratio changed from {ratio_in} to {ratio_out}')
         
-        out_name = os.path.splitext(im_name)[0] + f'_resized_{W}x{H}' + '.png'
```

Comment:
1. this would replace the original image if the input and output folder is same
2. it is beneficial to rename the image using new image sizes,m which gives user some traceability